src/new/wallet.js

```diff
@@ -2488,6 +2489,10 @@ class HathorWallet extends EventEmitter {
     });
 
     if (!tx.success) {
+      if (get(tx, 'message', '') === 'Transaction not found') {
```

Comment:
The same as in the other comment, we should have a method for that. In case we want to improve/change this validation somehow in the future.

Reply:
Created a guard method in the last commits, can you check?